### PR TITLE
feat(tack-menue): add track page to navbar

### PR DESCRIPTION
**Pull request by zerubeus**
# بسم الله الرحمن الرحيم

## What's new in this PR?

fixes

لله الحمد والمنة

### CODE REVIEW

File: src/components/molecules/AppNavBar/AppNavBar.tsx [hunk 1]
```diff
@@ -39,6 +113,9 @@ export default function AppNavBar({ authUser }: Proptypes) {
           {authUser && (
             <Fragment>
               <div className={classes.sectionDesktop}>
+                <Button startIcon={<MenuBookIcon />} onClick={() => navigate('/app')}>
+                  courses
```
Comment (abumalick) [inline src/components/molecules/AppNavBar/AppNavBar.tsx]: Well, technically it is `Tracks`

we have Tracks > Courses > Chapters

File: src/components/molecules/AppNavBar/AppNavBar.tsx [hunk 2]
```diff
@@ -13,18 +13,92 @@ import { ReactComponent as LogoFr } from '../../../assets/images/logo-horizontal
...
+import MenuBookIcon from '@material-ui/icons/MenuBook'
+import Button from '@material-ui/core/Button'
 
 type Proptypes = {
   authUser: FirebaseUser
 }
 
 export default function AppNavBar({ authUser }: Proptypes) {
+  const [mobileMoreAnchorEl, setMobileMoreAnchorEl] = useState<null | HTMLElement>(null)
+  const [anchorEl, setAnchorEl] = useState<null | HTMLElement>(null)
```
Comment (abumalick) [inline src/components/molecules/AppNavBar/AppNavBar.tsx:28]: the anchor logic is from material-ui documentation? It is the first time that I see that we use this for a menu logic.

**Review by abumalick — APPROVED** [review]
LGTM, good work 
:+1:

File: src/components/molecules/AppNavBar/AppNavBar.tsx (hunk 2 above)
Comment (zerubeus) [reply src/components/molecules/AppNavBar/AppNavBar.tsx:28]: Yes it is basically a copy past from mu

Comment (abumalick) [reply src/components/molecules/AppNavBar/AppNavBar.tsx:28]: okay, they probably have a good reason then hihi

**Pull request merged**